test/base/test_itensor.jl

```diff
@@ -316,16 +316,16 @@ end
       @test ndims(A) == 0
     end
 
-    @testset "trace (tr)" begin
+    @testset "trace (tr)" for ElType in (Float32, Float64, ComplexF32, ComplexF64)
```

Comment:
```suggestion
    @testset "trace (tr) (eltype=$elt)" for elt in (Float32, Float64, Complex{Float32}, Complex{Float64})
```
since that is the style I've been using elsewhere.